tests/daemon/mock-cache-size-provider.c

```diff
@@ -0,0 +1,40 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*- */
+
+/* Copyright 2017 Endless Mobile, Inc. */
```

Comment:
Probably needs an actual licence header, though if this is the style in the rest of the code, maybe not worth worrying about.

Reply:
Oh, derp, its literally 3 lines below. Too hasty to comment!